qucs/AUTHORS

```diff
@@ -17,12 +17,29 @@
 -- Boston, MA 02110-1301, USA.  
 --
 
-Maintainer:
-	Michael Margraf <michael.margraf@alumni.tu-berlin.de>
+Original developers:
+	Michael Margraf
+	Stefan Jahn
 
-Authors: (see THANKS file for details)
-	Michael Margraf <michael.margraf@alumni.tu-berlin.de>
-	Stefan Jahn <stefan@lkcc.org>
-	Jens Flucke <jens.flucke@gmx.de>
-	Gopala Krishna A <krishna.ggk@gmail.com>
-	Gunther Kraut <gn.kraut@online.de>
+Current developers:
+	Guilherme Brondani Torri
+	Mike Brinson
+	Richard Crozier
+	Bastien Roucaries
+	Frans Schreuder
+	Vadim Kuznetsov
+	Claudio Girardi
+	Felix Salfelder
+	Andr�s Mart�nez Mera
```

Comment:
...yep, I saw them but the strange thing is that if you then click on "View" (top right) GitHub displays the content properly, so they are not assuming the same encoding everywhere (?)

I have the terminal encoding to Latin1 here, so it looked fine to me :grin:
Switching to UTF-8 you get the characters above.

Should we use just 7-bit ASCII in text files like this?